client/lib/plugins/utils.js

```diff
@@ -161,7 +161,7 @@ export function normalizePluginData( plugin, pluginData ) {
 				returnData.author_name = extractAuthorName( item );
 				returnData.author_url = plugin.author_url || extractAuthorUrl( item );
 				break;
-			case 'sections':
+			case 'sections': {
```

Comment:
This change is unrelated to the PR, but needed since ESLint was complaining about `cleanItem` being defined in a `case` clause (see [`no-case-declarations`](https://eslint.org/docs/rules/no-case-declarations)).